pbs/pbsrequest.go

```diff
@@ -127,6 +135,17 @@ func ParsePBSRequest(r *http.Request, cache cache.Cache) (*PBSRequest, error) {
 	if pbsReq.Device == nil {
 		pbsReq.Device = &openrtb.Device{}
 	}
+	if pbsReq.SDK == nil {
+		pbsReq.SDK = &SDK{}
+	}
+	if pbsReq.SDK.Version != "0.0.1" {
+		if pbsReq.PBSUser != nil {
```

Comment:
The code isn't specific to the AN adapter actually. The issue here is that we used to pass the User gender field from Prebid Mobile (in v0.0.1) as an integer, and for Facebook demand they require the User object, where the gender field in OpenRTB is a string. When i implemented adding the User object here for Facebook demand (https://github.com/prebid/prebid-server/pull/94), I also updated Prebid Mobile to pass a string and we released that version as 0.0.2.

However, we recently realized we needed to also support the older version of Prebid Mobile v0.0.1 where we were passing an integer. This PR's purpose is to allow User gender to be passed as either a string or an integer to Prebid Server without getting a parsing error. Very open to a better way to solve this problem.